packages/test/test-service-load/src/loadTestDataStore.ts

```diff
@@ -233,16 +233,10 @@ export class LoadTestDataStoreModel {
 		this.taskId = `op_sender${config.runId % halfClients}`;
 		this.partnerId = (this.config.runId + halfClients) % this.config.testConfig.numClients;
 		const changed = (taskId) => {
-			this.deferUntilConnected(
-				() => {
-					if (taskId === this.taskId && this.taskStartTime !== 0) {
-						this.dir.set(taskTimeKey, this.totalTaskTime);
-						this.taskStartTime = 0;
-					}
-				},
-				(error) =>
-					this.logger.sendErrorEvent({ eventName: "TaskManager_OnValueChanged" }, error),
-			);
+			if (taskId === this.taskId && this.taskStartTime !== 0) {
```

Comment:
I'm not sure I agree with a direction in the PR.
We know that telemetry actually captures wrong-doing - it maybe not due to your changes, but it does good stuff (brings our attention to the problem). And we identified already one case that we can fix.
I'd love to see addressing it and re-evaluating if we can undo changes in this file but keep an error as an error.
At least for starters. And re-evaluate.

Basically, we have a problem, and having errors helps keep the problem active.